swiftglue/Makefile

```
@@ -37,17 +37,27 @@ $(3)_SDK:=$$(shell xcrun --sdk $(2) --show-sdk-path)
 
 ## debug
 $(3)_DEBUG_OUTPUTDIR:=$(BINDIR)/Debug/$(1)/$$($(3)_ARCH)
-$$($(3)_DEBUG_OUTPUTDIR)/$(MODULENAME): Makefile $(wildcard *.swift)
+$(3)_DEBUG_COUTPUTDIR:=$(BINDIR)/Debug/$(1)/$$($(3)_ARCH)/ofiles
+$$($(3)_DEBUG_COUTPUTDIR)/%.o : %.c
+	mkdir -p $$($(3)_DEBUG_COUTPUTDIR)
```

This makes the output nicer/quiet:

```suggestion
	$$(Q) mkdir -p $$($(3)_DEBUG_COUTPUTDIR)
```